src/storage/src/mem_table.rs

```diff
@@ -47,6 +47,7 @@ pub enum KeyOp {
 pub struct MemTable {
     pub(crate) buffer: BTreeMap<Bytes, KeyOp>,
     pub(crate) is_consistent_op: bool,
+    pub(crate) kv_size: usize,
```

Comment:
Use https://github.com/risingwavelabs/risingwave/blob/yuhao/over_win_expr/src/common/src/estimate_size/mod.rs#L190 instead of usize